Neos.ContentRepository/Classes/Domain/Context/Dimension/ContentDimensionZookeeper.php

```diff
@@ -0,0 +1,93 @@
+<?php
+namespace Neos\ContentRepository\Domain\Context\Dimension;
+
+/*
+ * This file is part of the Neos.ContentRepository package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+use Neos\ContentRepository\Domain;
+use Neos\Flow\Annotations as Flow;
+
+/**
+ * The content dimension zookeeper
```

Comment:
Internal factory of the AllowedDimensionSubspace; handles the logic of finding all available dimension combinations.

Purely internal.